ScarletEngine/Sources/UI/Public/Widgets.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+#include "Transform.h"
+
+namespace ScarletEngine::Widgets
+{
+    void DrawVec3Input(const char* Label, glm::vec3& Vec);
```

Comment:
Is it better to take a ref of a glm::vec3 ?

Reply:
This function may modify the value of the vector. It's used to draw the input panel for the vector.